### ISSUE TITLE
Fail to reproduce the training result on SNLI

### ISSUE THREAD

**Issue by ZacharyChenpk** [ShannonAI/Self_Explaining_Structures_Improve_NLP_Models#13]
I have tried the training script in this repository to train a model for SNLI dataset, and found that when training process continued, the predicted result of all valid cases seemed to fall into the same majority class, which made the accuracy fell to 0.3382. What should I do to reproduce the results in the paper?
